### DISCUSSION TITLE
Toast bug in Figma

### DISCUSSION THREAD

**Discussion by char-mainewang** [twilio-labs/paste#3446] [Help: Figma] [answered]
I can't seem to edit the text when the toast component is in a frame using Evergreen. if i edit it outside of the frame, it changes back to the default placeholder when i drag it back into the frame

https://github.com/twilio-labs/paste/assets/87334135/c120afd8-12c0-454f-8466-c4ce803f2430

**Comment by serifluous** [comment]
Hmmm...I think I know the reason. We might need to finally separate different text styles into their own layers for Toast and Alert.

I'll get it updated in our library, and hopefully will be able to get it published by early next week—We've been facing some publishing issues, and Figma's supposed to release a fix later this week.

**Reply by char-mainewang** [reply]
Gotcha, I'll keep an eye out for the update. Thank you for your help!

**Reply by serifluous** [reply, marked as answer]
Just published an update to fix this!
